views/team.hbs

```diff
@@ -34,67 +34,96 @@
-                            <div class="header-rotated-wrapper">{{#if this.teamPath}}<img src="{{this.teamPathRotated}}" alt="{{this.departmentName}}" class="header-rotated mr30">{{/if}}
+                    {{#if_even @index}}
+                        <div class='container-fluid section black-tab-container'>
+                            <div>
+                                <div>
+                                    <h1>{{this.departmentName}}</h1>
+                                </div>
                             </div>
-                            <div style="width: 100%;">
-                                <h4 class='lead'>Lead: {{this.teamLead}}</h4>
-                                <div class='individual-team__teammates-container'>
+                            <h4 class='Description lead-1'>Lead: {{this.teamLead}}</h4>
+                            <div class='individual-team__teammates-container'>
+                                {{#if this.isFourWide}}
                                     {{#each this.teammates}}
-                                        <div class='individual-team__teammate'>
+                                        <div class='individual-team__teammate four-wide'>
                                             <div class='pic'>
                                                 <img class='team-photo' src='{{this.picturePath}}' />
                                             </div>
-                                            <h4 class='Description'><b>{{this.name}}</b></h4>
+                                            <h4 class='Description'>{{this.name}}</h4>
                                         </div>
                                     {{/each}}
-                                </div>
+                                {{else}}
+                                    {{#each this.teammates}}
+                                        <div class='individual-team__teammate three-wide'>
+                                            <div class='pic'>
+                                                <img class='team-photo' src='{{this.picturePath}}' />
+                                            </div>
+                                            <h4 class='Description'>{{this.name}}</h4>
+                                        </div>
+                                    {{/each}}
+                                {{/if}}
+                            </div>
+                        </div>
+                {{else}}
+                    <div class='container-fluid section white-tab-container'>
+                        <div>
+                            <div>
+                                <h1>{{this.departmentName}}</h1>
                             </div>
                         </div>
+                        <h4 class='Description lead-2'>Lead: {{this.teamLead}}</h4>
+                        <div class='individual-team__teammates-container'>
+                            {{#if this.isFourWide}}
```

Comment:
I think you can get rid of some code repetition here. Instead of the if and else at the top level, you could do something like: `class="team-photo {{#if this.isFourWide}four-wide{{else}}three-wide{{/if}}` so you have an if statement for just that class.

Reply:
I wanted to write it like that, however since that line is inside the `{{#each this.teammates}}`, `this` ends up referring to the teammate, meaning that if I wanted it to work like that I'd have to add the `isFourWide` property to all of the individual teammates. If there's an easy way to get the parent of `this` that I'm missing, then it could be condensed a bit.

Reply:
However, I can take out a huge chunk of code bc I didn't realize I could use inline if's. Thanks :))